beacon-chain/rpc/nodev1/node_test.go

```diff
@@ -263,8 +263,8 @@ func TestListPeers(t *testing.T) {
 		expectedId := ids[0]
```

Comment:
Behavior is the same but the reason this is changed is because the old code was using `go_grpc_gateway_library` as a bazel target when it should have been using `go_default_library`